pallets/clusters/src/lib.rs

```diff
@@ -0,0 +1,476 @@
+//! This pallet `clusters` performs logic related to **Clusters**.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+pub use pallet::*;
+use sp_clamor::Hash128;
+use sp_core::bounded::BoundedVec;
+use sp_std::{vec, vec::Vec};
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(test)]
+mod dummy_data;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+/// The **settings** of a **Role**
+#[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, scale_info::TypeInfo)]
+pub struct RoleSetting {
+	/// Name of the setting
+	pub name: Vec<u8>,
```

Comment:
we decided to refactor bounded vectors after?
Maybe we should start here though (in the scope of clusters)?

Reply:
As discussed, `BoundedVec` now are used only in the extrinsics input, not in storage nor in structs.